group.py

```diff
@@ -2,4 +2,18 @@
 
 # Your code to go here...
 
-my_group =
+my_group = { "Jill" : { "Age": 26, "Job": ["Biologist"], "Connections": [{"Zalika" : "friend", "John": "Partner"}] 
```

Comment:
This looks very neat - it's great that you considered that people might have multiple jobs by making the "job" field a list of strings.

Reply:
Thank you for the feedback :)